README.md

```diff
@@ -59,6 +59,38 @@ pre-build packages, check the Downloads associated with
 the stable releases on the
 [github release page](https://github.com/pmem/pmdk/releases).
 
+### Pre-build Packages Windows ###
+
+The recommended and easiest way to install PMDK on Windows is
+to use a Microsoft vcpkg. Vcpkg is an open source tool and ecosystem
+created to libraries management.
+
+To install and link PMDK latest release into your Visual Studio solution
+at first you need to clone and set up vcpkg on your machine as described
+on the [vcpkg github page](https://github.com/Microsoft/vcpkg) in **Quick Start** section.
+
+In brief:
+
+```
+	> git clone https://github.com/Microsoft/vcpkg
+	> cd vcpkg
+	> .\bootstrap-vcpkg.bat
+	> .\vcpkg integrate install
+	> .\vcpkg install pmdk:x64-windows
+```
+
+Last command can take a while - it is PMDK building and installing time.
+
+After successfully completing all above steps libraries are ready to
+include and use in Visual Studio, no additional configuration is required.
+Just open VS with your already existing project or create new one
+(remember to use platform **x64**) and then include PMDK libraries in that way:
+
+```
+	#include <libpmem.h>
+	#include <libpmemobj.h>
+```
```

Comment:
I don't believe this example is useful. There is nothing Windows specific in it.